src/pages/partners/affiliate-ib/_deriv-ib-programme.tsx

```diff
@@ -143,21 +200,54 @@ const DerivIBProgramme = () => {
                         <Localize translate_text="_t_Our introducing broker programme is available to all Deriv affiliates._t_" />
                     </SubtitleHeader>
                     <SubtitleHeader as="p" type="sub-section-title" align="center" weight="normal">
-                        <Localize translate_text="_t_Earn commission from your clients’ trades on Deriv MT5._t_" />
+                        <Localize translate_text="_t_Earn commission from your clients’ trades on Deriv MT5 and Deriv X._t_" />
                     </SubtitleHeader>
-                    <Header as="h2" mb="2rem" type="page-title" mt="4rem" align="center">
-                        <Localize translate_text="_t_Deriv MT5_t_" />
-                    </Header>
                 </TitleWrapper>
+                <TabsContainer>
+                    {is_mounted && ['mt5', 'x'].map((tabName: ActiveTabType) => (
+                        <Item
+                            key={tabName}
+                            onClick={() => handleTabClick(tabName)}
+                            active_tab={activeTab}
+                            name={tabName}
+                        >
+                            <Header as="h4">
+                                <Localize translate_text={`_t_Deriv ${tabName.toUpperCase()}_t_`} />
+                            </Header>
+                        </Item>
+                    ))}
+                </TabsContainer>
                 <IBSectionContainer padding="4rem 0 9.6rem 0">
-                    <StyledHeaderCommission as="h3" type="main-paragraph" mb="1.6rem">
-                        <Localize translate_text="_t_Choose a commission plan:_t_" />
-                    </StyledHeaderCommission>
-                    <StyledCardWrapper>
-                        <DERIVIBDMT5Cards data={ib_dmt5_synthetic} />
-                        <DERIVIBDMT5Cards data={ib_dmt5_financial} />
-                        <DERIVIBDMT5Cards data={ib_dmt5_financialSTP} />
-                    </StyledCardWrapper>
+                    {activeTab === 'mt5' && (
+                        <>
+                            <Header as="h2" mb="3rem" type="page-title" align="center">
```

Comment:
HI @eduard-deriv 
                    `<>
                            <Header as="h2" mb="3rem" type="page-title" align="center">
                                <Localize translate_text="_t_Deriv MT5_t_" />
                            </Header>
                            <StyledHeaderCommission as="h3" type="main-paragraph" mb="1.6rem">
                                <Localize translate_text="_t_Choose a commission plan:_t_" />
                            </StyledHeaderCommission>
                            <StyledCardWrapper>
                                <DERIVIBDMT5Cards data={ib_dmt5_synthetic} />
                                <DERIVIBDMT5Cards data={ib_dmt5_financial} />
                                <DERIVIBDMT5Cards data={ib_dmt5_financialSTP} />
                            </StyledCardWrapper>
                        </>`
                        
    As this whole block is same and you are using it for `x` as well then isn’t it better to have a different component ?

Reply:
if you look closely, the differences are not only in X and MT5, but also in header, headerCommission, number of cards and other. Yes, i can create different component with lots of props but this is not necessary. Thanks